<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Models</class>
 <widget class="QDialog" name="Models">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="variables_label">
     <property name="text">
      <string>Custom Project Variables</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="SpaceCheckListView" name="variables_view" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="SpaceCheckListView" name="layouts_view" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="layouts_label">
     <property name="text">
      <string>Print Layouts (exported as templates)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="description">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Additive project settings, that should be exported (if available). Like Map Themes, Custom Variables and Print Layouts.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="mapthemes_label">
     <property name="text">
      <string>Map Themes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="SpaceCheckListView" name="mapthemes_view" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="validatortopping_checkbox">
     <property name="text">
      <string>Convert validation configuration file to topping file</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpaceCheckListView</class>
   <extends></extends>
   <header>QgisModelBaker.utils.gui_utils</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
